lib/server.js

```diff
@@ -175,7 +183,8 @@ function handleRequest (opts, req, resp) {
             query: urlData.query,
             method: req.method.toLowerCase(),
             headers: req.headers,
-            body: body
+            body: body,
+            reqId: reqId
```

Comment:
Soooo we're back to the previous version in which all of the headers had the request ID set?